fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -1796,6 +1800,37 @@ private void handleShowCatalogs() {
         resultSet = new ShowResultSet(showCatalogsStmt.getMetaData(), rowSet);
     }
 
+    // show warehouse statement
+    private void handleShowWarehouses() throws AnalysisException, DdlException {
+        ShowWhStmt showStmt = (ShowWhStmt) stmt;
```

Comment:
I think we should add test case for `create warehouse` and `show warehouses`, add all the test case in the end may not be a good idea.